pmm-app/src/pmm-inventory/panel.tsx

```diff
@@ -94,6 +94,11 @@ export const nodesColumns = [
     dataIndex: 'node_id',
     key: 'node_id',
   },
+  {
+    title: 'Node Type',
```

Comment:
looks like something wrong here. Title is `Node Type`, but key and dataIndex are `agentType`

Reply:
This is just a common name for this type of column, all tables have similar structure

Reply:
maybe it's better to rename it then. looks really confusing.